MediaBrowser.Common/Plugins/IPluginRegistrar.cs

```diff
@@ -0,0 +1,17 @@
+namespace MediaBrowser.Common.Plugins
+{
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// Defines the <see cref="IPluginRegistrar" />.
+    /// </summary>
+    public interface IPluginRegistrar
```

Comment:
I think this is a better name.
```suggestion
    public interface IPluginServiceRegistrator
```